### PR TITLE
[Release 1.3.10] Onboard test manifests and add remaining components

### PR DESCRIPTION
**Pull request by prudhvigodithi**
### Description
Onboard test manifests and add remaining components.
NOTE: missing CCR due to pending version increment PR https://github.com/opensearch-project/cross-cluster-replication/pull/743

### Issues Resolved
Part of https://github.com/opensearch-project/opensearch-build/issues/3331

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: manifests/1.3.10/opensearch-dashboards-1.3.10-test.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  - name: OpenSearch-Dashboards
+    bwc-test:
+      test-configs:
+        - with-security
+        - without-security
+  - name: functionalTestDashboards
+    integ-test:
+      test-configs:
+        - with-security
+        - without-security
```
Comment (gaiksaya) [inline manifests/1.3.10/opensearch-dashboards-1.3.10-test.yml:17]: We have changed the way we run dashboards test run so this should change to per component. @Divyaasm can add more.

Comment (prudhvigodithi) [reply manifests/1.3.10/opensearch-dashboards-1.3.10-test.yml:17]: Any example? @gaiksaya @Divyaasm

**Comment by prudhvigodithi** [comment]
Added cross-cluster-replication as the existing version increment PR is merged.
https://github.com/opensearch-project/cross-cluster-replication/pull/743

File: manifests/1.3.10/opensearch-dashboards-1.3.10-test.yml (hunk 1 above)
Comment (gaiksaya) [reply manifests/1.3.10/opensearch-dashboards-1.3.10-test.yml:17]: Example: https://github.com/Divyaasm/opensearch-build/blob/ghissue/manifests/2.6.0/opensearch-dashboards-2.6.0-test.yml
Not sure if the dashboards repo had backported the running in parallel part for 1.3 branch.

Comment (prudhvigodithi) [reply manifests/1.3.10/opensearch-dashboards-1.3.10-test.yml]: Thanks, referring to that link and from what I understood we only need to add the components `opensearch-dashboards-1.3.10.yml` except for the `functionalTestDashboards`. Updated the change in my latest PR. However the backport of this PR https://github.com/opensearch-project/opensearch-dashboards-functional-test/pull/644 should be done to `1.3` to run the OSD integ tests.

**Review by gaiksaya — APPROVED** [review]

File: manifests/1.3.10/opensearch-dashboards-1.3.10-test.yml (hunk 1 above)
Comment (gaiksaya) [reply manifests/1.3.10/opensearch-dashboards-1.3.10-test.yml]: Right! This PR needs to be backported too https://github.com/opensearch-project/opensearch-dashboards-functional-test/pull/234

**Comment by prudhvigodithi** [comment]
Noticed one more change with CI image from `v1` to opensearchstaging/ci-runner:ci-runner-rockylinux8-opensearch-dashboards-integtest-v2`, updated in my latest commit.

**Pull request merged**